src/components/MenuItem.tsx

```diff
@@ -451,7 +453,7 @@ function MenuItem(
                                                     imageStyles={[styles.alignSelfCenter]}
                                                     size={CONST.AVATAR_SIZE.DEFAULT}
                                                     source={icon as AvatarSource}
-                                                    fallbackIcon={fallbackIcon}
+                                                    fallbackIcon={fallbackIcon ?? illustrations.FallbackAvatar}
```

Comment:
Should we move the fallback icon including the backupt with the nullish-coalescing (`??`) to a separate variable? I think that's cleaner in general when it comes to "theme-dependent default props"

```ts
function MenuItem({
  // ...
  fallbackIcon: fallbackIconProp
}) {
  // ...
  const fallbackIcon = fallbackIconProp ?? illustrations.FallbackAvatar

  // return (...)
}
```